spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'foods page' do
```

Comment:
We will also want to test that we have a list of only 10 foods that contain the ingredient you searched for. And for each of those foods we should see:
```
- The food's GTIN/UPC code
- The food's description
- The food's Brand Owner
- The food's ingredients
```